app/views/shared/_nav-links.html.erb

```diff
@@ -0,0 +1,20 @@
+<nav class="nav-main d-flex flex-wrap items-center justify-content-between">
+
+  <div class="nav-links-extra d-flex flex-start align-items-center mt-2 mr-3">
+    <% unless current_page?(root_path) %>
+      <%= link_to 'FOOD', root_path, class: 'text-dark pb-2' %>
+      <%= link_to 'DRINKS', root_path, class: 'text-dark pb-2' %>
+      <%= link_to 'CONVENIENCE', root_path, class: 'text-dark pb-2' %>
+    <% end %>
+  </div>
+
+  <div>
+    <% if user_signed_in? %>
+      <%= link_to "LOG OUT", destroy_user_session_path, method: :delete, class: 'btn btn-dark rounded-pill ml-3 px-2 px-md-4' %>
+    <% else %>
+      <%= link_to 'LOG IN', new_user_session_path, class: 'btn btn-outline-dark mr-2 mr-md-3 rounded-pill px-2 px-md-4' %>
+      <%= link_to 'SIGN UP', new_user_registration_path, class: 'btn btn-dark rounded-pill px-2 px-md-4' %>
+    <% end %>
```

Comment:
\<div class="d-none" id="current-user-email">
          <%= current_user.email %>
        \</div>
Request Chris to add this after line 17, I am inserting user email as a no-display field, I need this element for my JS which redirects to devise sign-in, if the user has clicked "Add to Cart" but is not signed in .... thanks !

Reply:
Hey @saikatb221062, Ive added the requested code and pushed to the branch.